src/vfb_query_builder/query_roller.py

```diff
@@ -716,9 +718,9 @@ def term_info_export(escape=True):
         # This whole approach feels a bit hacky...
         qf = getattr(ql, qm)
         q_name = qf.__kwdefaults__['q_name']
-        q = qf(short_form=['$ID'])
+        q = qf(short_form='[$id]')
         if escape:
-            out[q_name] = saxutils.escape(q)
+            out[q_name] = '&quot;statement&quot;: &quot;' + q.replace('  ',' ').replace('<','&lt;').replace('\n',' ').replace('  ',' ') + '&quot;, &quot;parameters&quot; : { &quot;id&quot; : &quot;$ID&quot; }'
         else:
```

Comment:
OK with this as long as it works.   Curious to see if we can get a more generic fix, but no big deal.